docs/dl-run-first-contract.md

```diff
@@ -0,0 +1,69 @@
+# Run your first contract
+
+## Before you begin
+
+You should have already downloaded the `scalardl-client-sdk` directory and registered your certificate. If you haven't ... **add links to other docs here**
+
+## Run the StateUpdater contract
+
+We will run the contract [`scr/main/java/com/org1/contract/StateUpdater.java`](https://github.com/scalar-labs/scalardl-client-sdk/blob/master/src/main/java/com/org1/contract/StateUpdater.java).
+
+In the `scalardl-client-sdk` directory:
+
+1. Compile the contract
+
+    ```
+    $ ./gradlew assemble
+    ```
+
+    This will generate `build/classes/java/main/com/org1/contract/StateUpdater.class`.
+
+2. Register the contract
+
+    ```
+    $ client/bin/register-contract -properties conf/client.properties -contract-id StateUpdater -contract-binary-name com.org1.contract.StateUpdater -contract-class-file build/classes/java/main/com/org1/contract/StateUpdater.class
+    ```
+
+    The `contract-id` is set by the user. Above we have chosen `StateUpdater`, but you can choose anything you like. You can set different contract IDs on the same contract to clarify "who did what" in a tamper-evident way. For example, consider a voting application. In the application, anyone can vote with the same voting logic, and hence can use the same Contract, but A's vote and B's vote need to be properly and securely distinguished; A cannot vote for B, and vice versa. This can be achieved by having different contract IDs on the same contract.
+
+3. Execute the contract
+
+    ```
+    $ client/bin/execute-contract -properties conf/client.properties -contract-id StateUpdater -contract-argument '{"asset_id": "abcdef", "state": 3}'
```

Comment:
I think brief explanation about `asset` in Scalar DL is needed here before this, otherwise the user just copy it and works but don't really understand what he is doing.